hudi-client/hudi-client-common/src/main/java/org/apache/hudi/table/action/rollback/MarkerBasedRollbackStrategy.java

```diff
@@ -159,9 +161,18 @@ protected HoodieRollbackRequest getRollbackRequestForAppend(String markerFilePat
       HoodieLogFile latestLogFile = latestLogFileOption.get();
       // NOTE: Marker's don't carry information about the cumulative size of the blocks that have been appended,
       //       therefore we simply stub this value.
-      logFilesWithBlocksToRollback = Collections.singletonMap(latestLogFile.getPath().toString(), -1L);
+      FileSystem fileSystem = table.getMetaClient().getFs();
```

Comment:
We need to ensure we merge multiple rollback requests such that, we have just 1 rollbackRequest per file slice. 
eventually, during execution we will add rollback blocks once per roll back request. So, lets not break that assumption.